Libraries/Lists/__tests__/SectionList-test.js

```diff
@@ -26,6 +26,16 @@ describe('SectionList', () => {
     );
     expect(component).toMatchSnapshot();
   });
+  it('renders empty list with empty component', () => {
```

Comment:
I wouldn't worry about duplicating these empty cases in FlatList and SectionList - just VirtualizedList should be fine. Adding to the "bells and whistles" cases is good though.